dcm4che-imageio-test/pom.xml

```diff
@@ -290,6 +290,11 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.dcm4che</groupId>
+      <artifactId>dcm4che-imageio</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```

Comment:
I have added this dependency explicitly, because only that way Jacoco generates a correct aggregated code coverage report for the classes in the dcm4che-imageio.
This increases the report code coverage for these changes from 45.5% to 90.9%.
https://sonarcloud.io/dashboard?id=org.dcm4che%3Adcm4che-parent&pullRequest=1004